### DISCUSSION TITLE
Feature Request: Show only working hours for deadlines in tasks

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/Paid-plugins#179] [Questions & Answers]
Hello,
it would be amazing to define somewhere the working hours and then instantly jump to working hours to not always to have to scroll so much, like when the times start at midnight.
![Monosnap-Kimai – Zeiterfassung-2024-07-15-09-02-30](https://github.com/user-attachments/assets/5a78e132-ead7-4dfd-9348-7fa4a9fd3abb)

Best wishes,

Niels

**Comment by kevinpapst** [comment]
Has nothing to do with tasks. And there is already a feature request existing, so I close this one as duplicate.
